gen_readme.py

```diff
@@ -39,6 +39,25 @@
         README += "#### "
         README += f"[{mod['name']}]({mod['sourceLocation']})"
 
+        # mod must have versions
+        if mod["versions"] is None or len(mod["versions"]) == 0:
+            continue
+
+        # mod must have a non-vulnerable version
+        all_vulnerable = True
+        for key, version in mod["versions"]:
+            if version.flags is not None:
```

Comment:
Wouldn't this have a mod without any flags ever being set be defined as having all vulnerable?

Reply:
Whoops... fixed in  03ac8e90